### PR TITLE
Remove the old .deb files we were carrying around

### PR DESCRIPTION
**Pull request by lbonn**

### CODE REVIEW

File: tests/test_install_aktualizr_and_update.sh [hunk 1]
```diff
@@ -33,4 +36,4 @@ if [ $? -ne 0 ];then
     echo "ERROR: $filename does not exist or sha256sum does not match."
```
Comment (Zee314159) [inline tests/test_install_aktualizr_and_update.sh]: I also have a bit of concern at line33-line35. what if somehow `aktualizr_info` run into some error and return a blank filename, and `ls $TEMP_DIR/images/$filename` still return to 0, but actually it shouldn't?

Comment (lbonn) [reply tests/test_install_aktualizr_and_update.sh]: yes actually this test is weird, it should just be `if [ ! -f ... ]; then`

File: tests/test_install_aktualizr_and_update.sh [hunk 2]
```diff
@@ -33,4 +36,4 @@ if [ $? -ne 0 ];then
...
     exit 1
 fi
-
+rm -rf $TEMP_DIR
```
[the remark this answers is not in the document]
Comment (lbonn) [reply tests/test_install_aktualizr_and_update.sh]: yes

**Comment by lbonn** [comment]
Hm I've refactored some bash line into two steps and now it fails because jq is missing on our docker envs.

Looks like the tests passed by mistake before that.

**Pull request merged**